spring-cloud-sleuth-stream/src/main/java/org/springframework/cloud/sleuth/stream/SleuthStreamProperties.java

```diff
@@ -44,4 +45,40 @@ public void setEnabled(boolean enabled) {
 	public void setGroup(String group) {
 		this.group = group;
 	}
+
+	public Poller getPoller() {
+		return this.poller;
+	}
+
+	public void setPoller(Poller poller) {
+		this.poller = poller;
+	}
+
+	public static class Poller {
+		/**
+		 * Fixed delay (ms). Default: 1000
+		 */
+		private long fixedDelay = 1000L;
```

Comment:
Javadoc links don't work in `@ConfigurationProperties`, so please don't, actually.